packages/board/src/vaadin-board-row.js

```diff
@@ -255,6 +251,19 @@ class BoardRow extends ResizeMixin(ElementMixin(PolymerElement)) {
     }
   }
 
+  /** @private */
+  _shouldRecalculate(width, breakpoints) {
+    // Should not recalculate if row is invisible
+    if (isElementHidden(this)) {
```

Comment:
It seems that `isElementHiddenDirectly` would be more appropriate in this case as `isElementHidden` is typically used to know if the element is hidden due to a hidden ancestor.

```suggestion
    if (isElementHidden(this)) {
```

Reply:
But the issue is about the whole board, or one of its ancestors, being hidden, not the row itself?

Reply:
Oh, you are totally right. It got all mixed up in my head. I'd suggest adding a test that hides the board through an ancestor then.

Reply:
Added a test which toggles the visibility of the container div.